framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -636,6 +636,16 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
     }
 
     @Override
+    public List<AnnotationMirror> getJaifInferenceIgnoredAnnotations() {
+        List<AnnotationMirror> out = super.getJaifInferenceIgnoredAnnotations();
```

Comment:
This side-effects the value returned by the superclass.  So the superclass shouldn't return a value that exposes its representation.